gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -70,34 +81,47 @@ def run(argv=None):
   known_args, pipeline_args = vcf_to_bq_common.parse_args(argv,
                                                           _COMMAND_LINE_OPTIONS)
   options = pipeline_options.PipelineOptions(pipeline_args)
+  is_direct_runner = _is_direct_runner(beam.Pipeline(options=options))
   google_cloud_options = options.view_as(pipeline_options.GoogleCloudOptions)
-  # TODO(allieychen): Add support for local location.
-  if not google_cloud_options.temp_location or not google_cloud_options.project:
-    raise ValueError('temp_location and project must be set.')
-
+  if not google_cloud_options.project:
+    raise ValueError('project must be set.')
+  if not is_direct_runner and not known_args.output_file.startswith('gs://'):
+    raise ValueError('Please set the output file {} to GCS when running with '
+                     'DataflowRunner.'.format(known_args.output_file))
+  if is_direct_runner:
+    known_args.number_of_bases_per_shard = -1
```

Comment:
Sorry Allie: I now think it's better to use `sys.maxsize` instead of `-1` here. I was originally thinking that we'd want to avoid overflow, but given that we're special-casing that value anyway, then we'd need to create a proper constant for `-1` too, which is strange. We can use `maxsize` as a defined constant representing "infinite" range.